internal/collector/vector/run_script.go

```diff
@@ -11,6 +11,22 @@ VECTOR_DATA_DIR=%s
 echo "Creating the directory used for persisting Vector state $VECTOR_DATA_DIR"
 mkdir -p ${VECTOR_DATA_DIR}
 
+# Remap lowercase to uppercase if it's set
+if [ -z "${HTTP_PROXY:-}" ] && [ -n "${http_proxy:-}" ]; then
+  export HTTP_PROXY="$http_proxy"
+  echo HTTP_PROXY="$HTTP_PROXY"
```

Comment:
If we are going to add a "log" statement then lets say what we are doing... "Setting HTTP_PROXY=...."

Reply:
Is the proxy url actually something we want to put into a log?   Couldn't it contain user/pwd?